dht/i_partitioner.cc

```diff
@@ -290,8 +290,9 @@ ring_position_exponential_sharder::next(const schema& s) {
     return std::make_optional(std::move(ret));
 }
```

Comment:
changelog

Reply:
Next patch moves the sharders to separate header. I didn't want to move unused code but didn't want to silently remove it together with move so I split it into two commits. One removing unused code and second moving the rest to separate header. Will add the changelog.

Reply:
AFAIK this was used for range-scans, before that stateful range scan implementation, which doesn't use them anymore.

Reply:
Thanks @denesb